misc/qutebrowser.appdata.xml

```diff
@@ -41,8 +41,46 @@
 	<url type="bugtracker">https://github.com/qutebrowser/qutebrowser/issues/</url>
 	<url type="donation">https://github.com/qutebrowser/qutebrowser#donating</url>
 	<releases>
-		<release version="1.3.0" date="2018-05-04"/>
+        <release version="1.5.0" date="2018-10-03"/>
```

Comment:
note to self: Add 1.5.1 here by hand if it's released by the time I merge this :wink: